readme.md

```diff
@@ -25,6 +25,27 @@ normalizeUrl('HTTP://xn--xample-hva.com:80/?b=bar&a=foo');
 ```
 
 
+## API
+
+### normalizeUrl(str, [options])
+
+#### str
+
+*Required*  
+Type: `string`
+
+#### options.keepProtocol
```

Comment:
I think I would prefer `options.preserveRelativeProtocol` because it is more grammatically correct.

How about the inverse?

`options.normalizeProtocol` (default `true`, pass `false` for `keepProtocol` behaviour)